.github/workflows/robot/internal/bot/check.go

```diff
@@ -36,49 +38,107 @@ func (b *Bot) Check(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
-	if b.c.Review.IsInternal(b.c.Environment.Author) {
-		// Remove stale "Check" status badges inline for internal reviews.
-		err := b.dismiss(ctx,
```

Comment:
We used to run `dismiss` for internal reviews. It seems now we only run it for external reviews. Is that intentional?